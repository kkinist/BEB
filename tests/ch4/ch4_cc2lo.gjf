%chk=ch4.chk
%nprocs=4
%mem=1000mw
# CCSD(T,maxcyc=100)/GEN gfinput geom=check scf=xqc

BEB step 10: low-spin CCSD(T) for dication of ch4

2 1

C H 0
cc-pVTZ
****

