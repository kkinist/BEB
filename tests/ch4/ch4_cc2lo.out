 Entering Gaussian System, Link 0=g09
 Initial command:
 /usr/local/apps/g09/l1.exe "/wrk/irikura/Gau-28874.inp" -scrdir="/wrk/irikura/"
 Entering Link 1 = /usr/local/apps/g09/l1.exe PID=     28876.
  
 Copyright (c) 1988,1990,1992,1993,1995,1998,2003,2009,2013,
            Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 09 program.  It is based on
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 09, Revision D.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, B. Mennucci, 
 G. A. Petersson, H. Nakatsuji, M. Caricato, X. Li, H. P. Hratchian, 
 A. F. Izmaylov, J. Bloino, G. Zheng, J. L. Sonnenberg, M. Hada, 
 M. Ehara, K. Toyota, R. Fukuda, J. Hasegawa, M. Ishida, T. Nakajima, 
 Y. Honda, O. Kitao, H. Nakai, T. Vreven, J. A. Montgomery, Jr., 
 J. E. Peralta, F. Ogliaro, M. Bearpark, J. J. Heyd, E. Brothers, 
 K. N. Kudin, V. N. Staroverov, T. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. Rendell, J. C. Burant, S. S. Iyengar, J. Tomasi, 
 M. Cossi, N. Rega, J. M. Millam, M. Klene, J. E. Knox, J. B. Cross, 
 V. Bakken, C. Adamo, J. Jaramillo, R. Gomperts, R. E. Stratmann, 
 O. Yazyev, A. J. Austin, R. Cammi, C. Pomelli, J. W. Ochterski, 
 R. L. Martin, K. Morokuma, V. G. Zakrzewski, G. A. Voth, 
 P. Salvador, J. J. Dannenberg, S. Dapprich, A. D. Daniels, 
 O. Farkas, J. B. Foresman, J. V. Ortiz, J. Cioslowski, 
 and D. J. Fox, Gaussian, Inc., Wallingford CT, 2013.
 
 ******************************************
 Gaussian 09:  ES64L-G09RevD.01 24-Apr-2013
                17-May-2017 
 ******************************************
 %chk=ch4.chk
 %nprocs=4
 Will use up to    4 processors via shared memory.
 %mem=1000mw
 ---------------------------------------------------
 # CCSD(T,maxcyc=100)/GEN gfinput geom=check scf=xqc
 ---------------------------------------------------
 1/29=2,38=1/1;
 2/12=2,40=1/2;
 3/5=7,11=9,14=-4,16=1,24=10,25=1,30=1/1,2,3;
 4//1;
 5/5=2,8=3,13=1,38=5/2,8;
 8/6=4,9=120000,10=1/1,4;
 9/5=7,6=100,14=2/13;
 6/7=2,8=2,9=2,10=2/1;
 99/5=1,9=1/99;
 -------------------------------------------------
 BEB step 10: low-spin CCSD(T) for dication of ch4
 -------------------------------------------------
 Structure from the checkpoint file:  "ch4.chk"
 Charge =  2 Multiplicity = 1
 Redundant internal coordinates found in file.
 C,0,0.,0.000000001,0.000000001
 H,0,-0.000000006,-0.0000000007,1.0935113774
 H,0,0.,-1.0309724112,-0.3645037927
 H,0,-0.892848297,0.5154862096,-0.3645037952
 H,0,0.892848303,0.5154862062,-0.3645037855
 Recover connectivity data from disk.
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0        0.000000    0.000000    0.000000
      2          1           0        0.000000    0.000000    1.093511
      3          1           0        0.000000   -1.030972   -0.364504
      4          1           0       -0.892848    0.515486   -0.364504
      5          1           0        0.892848    0.515486   -0.364504
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.093511   0.000000
     3  H    1.093511   1.785697   0.000000
     4  H    1.093511   1.785697   1.785697   0.000000
     5  H    1.093511   1.785697   1.785697   1.785697   0.000000
 Stoichiometry    CH4(2+)
 Framework group  TD[O(C),4C3(H)]
 Deg. of freedom     1
 Full point group                 TD      NOp  24
 Largest Abelian subgroup         D2      NOp   4
 Largest concise Abelian subgroup D2      NOp   4
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0        0.000000    0.000000    0.000000
      2          1           0        0.631339    0.631339    0.631339
      3          1           0       -0.631339   -0.631339    0.631339
      4          1           0       -0.631339    0.631339   -0.631339
      5          1           0        0.631339   -0.631339   -0.631339
 ---------------------------------------------------------------------
 Rotational constants (GHZ):    157.2594344    157.2594344    157.2594344
 General basis read from cards:  (5D, 7F)
 AO basis set in the form of general basis input (Overlap normalization):
      1 0
 S   7 1.00       0.000000000000
      0.8236000000D+04  0.5419783203D-03
      0.1235000000D+04  0.4192873817D-02
      0.2808000000D+03  0.2152216205D-01
      0.7927000000D+02  0.8353432195D-01
      0.2559000000D+02  0.2395828457D+00
      0.8997000000D+01  0.4428528419D+00
      0.3319000000D+01  0.3517995618D+00
 S   6 1.00       0.000000000000
      0.2808000000D+03 -0.5949224937D-04
      0.7927000000D+02 -0.1148158310D-02
      0.2559000000D+02 -0.1001913745D-01
      0.8997000000D+01 -0.6121949230D-01
      0.3319000000D+01 -0.1732698541D+00
      0.3643000000D+00  0.1072915192D+01
 S   1 1.00       0.000000000000
      0.9059000000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1285000000D+00  0.1000000000D+01
 P   3 1.00       0.000000000000
      0.1871000000D+02  0.3942638716D-01
      0.4133000000D+01  0.2440889849D+00
      0.1200000000D+01  0.8154920089D+00
 P   1 1.00       0.000000000000
      0.3827000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.1209000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.1097000000D+01  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.3180000000D+00  0.1000000000D+01
 F   1 1.00       0.000000000000
      0.7610000000D+00  0.1000000000D+01
 ****
      2 0
 S   3 1.00       0.000000000000
      0.3387000000D+02  0.2549486323D-01
      0.5095000000D+01  0.1903627659D+00
      0.1159000000D+01  0.8521620222D+00
 S   1 1.00       0.000000000000
      0.3258000000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1027000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.1407000000D+01  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.3880000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.1057000000D+01  0.1000000000D+01
 ****
      3 0
 S   3 1.00       0.000000000000
      0.3387000000D+02  0.2549486323D-01
      0.5095000000D+01  0.1903627659D+00
      0.1159000000D+01  0.8521620222D+00
 S   1 1.00       0.000000000000
      0.3258000000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1027000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.1407000000D+01  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.3880000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.1057000000D+01  0.1000000000D+01
 ****
      4 0
 S   3 1.00       0.000000000000
      0.3387000000D+02  0.2549486323D-01
      0.5095000000D+01  0.1903627659D+00
      0.1159000000D+01  0.8521620222D+00
 S   1 1.00       0.000000000000
      0.3258000000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1027000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.1407000000D+01  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.3880000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.1057000000D+01  0.1000000000D+01
 ****
      5 0
 S   3 1.00       0.000000000000
      0.3387000000D+02  0.2549486323D-01
      0.5095000000D+01  0.1903627659D+00
      0.1159000000D+01  0.8521620222D+00
 S   1 1.00       0.000000000000
      0.3258000000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1027000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.1407000000D+01  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.3880000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.1057000000D+01  0.1000000000D+01
 ****

 There are    26 symmetry adapted cartesian basis functions of A   symmetry.
 There are    23 symmetry adapted cartesian basis functions of B1  symmetry.
 There are    23 symmetry adapted cartesian basis functions of B2  symmetry.
 There are    23 symmetry adapted cartesian basis functions of B3  symmetry.
 There are    23 symmetry adapted basis functions of A   symmetry.
 There are    21 symmetry adapted basis functions of B1  symmetry.
 There are    21 symmetry adapted basis functions of B2  symmetry.
 There are    21 symmetry adapted basis functions of B3  symmetry.
    86 basis functions,   120 primitive gaussians,    95 cartesian basis functions
     4 alpha electrons        4 beta electrons
       nuclear repulsion energy        13.3922467928 Hartrees.
 NAtoms=    5 NActive=    5 NUniq=    2 SFac= 4.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 1 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=    86 RedAO= T EigKep=  1.58D-03  NBF=    23    21    21    21
 NBsUse=    86 1.00D-06 EigRej= -1.00D+00 NBFU=    23    21    21    21
 ExpMin= 1.03D-01 ExpMax= 8.24D+03 ExpMxC= 2.81D+02 IAcc=1 IRadAn=         1 AccDes= 0.00D+00
 Harris functional with IExCor=  205 and IRadAn=       1 diagonalized for initial guess.
 HarFok:  IExCor=  205 AccDes= 0.00D+00 IRadAn=         1 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=     500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Initial guess orbital symmetries:
       Occupied  (A1) (A1) (T2) (T2)
       Virtual   (T2) (A1) (T2) (T2) (T2) (T2) (T2) (T2) (T2) (T2)
                 (T2) (A1) (E) (E) (A1) (T1) (T1) (T1) (T2) (T2)
                 (T2) (T2) (T2) (T2) (T2) (T2) (T2) (E) (E) (A1)
                 (T2) (T2) (T2) (A1) (T2) (T2) (T2) (E) (E) (T1)
                 (T1) (T1) (T2) (T2) (T2) (T1) (T1) (T1) (T2) (T2)
                 (T2) (E) (E) (A1) (T2) (T2) (T2) (E) (E) (T1)
                 (T1) (T1) (T2) (T2) (T2) (A1) (T2) (T2) (T2) (T2)
                 (T2) (T2) (T1) (T1) (T1) (A1) (E) (E) (T2) (T2)
                 (T2) (A1)
 Keep R1 ints in memory in symmetry-blocked form, NReq=9019840.
 Requested convergence on RMS density matrix=1.00D-08 within  64 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 Density matrix breaks symmetry, PCut= 1.00D-04
 Density has only Abelian symmetry.
 Density matrix breaks symmetry, PCut= 1.00D-07
 Density has only Abelian symmetry.
 Density matrix breaks symmetry, PCut= 1.00D-07
 Density has only Abelian symmetry.
 Density matrix breaks symmetry, PCut= 1.00D-07
 Density has only Abelian symmetry.
 Density matrix breaks symmetry, PCut= 1.00D-07
 Density has only Abelian symmetry.
 Density matrix breaks symmetry, PCut= 1.00D-07
 Density has only Abelian symmetry.
 Density matrix breaks symmetry, PCut= 1.00D-07
 Density has only Abelian symmetry.
 Density matrix breaks symmetry, PCut= 1.00D-07
 Density has only Abelian symmetry.
 Density matrix breaks symmetry, PCut= 1.00D-07
 Density has only Abelian symmetry.
 SCF Done:  E(RHF) =  -38.7885854479     A.U. after    8 cycles
            NFock=  8  Conv=0.84D-09     -V/T= 1.9812
 QCSCF skips out because SCF is already converged.
 ExpMin= 1.03D-01 ExpMax= 8.24D+03 ExpMxC= 2.81D+02 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 HarFok:  IExCor=  205 AccDes= 0.00D+00 IRadAn=         5 IDoV=-2 UseB2=F ITyADJ=14
 ICtDFT= 12500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 Range of M.O.s used for correlation:     2    86
 NBasis=    86 NAE=     4 NBE=     4 NFC=     1 NFV=     0
 NROrb=     85 NOA=     3 NOB=     3 NVA=    82 NVB=    82

 **** Warning!!: The largest alpha MO coefficient is  0.12660609D+02

 Semi-Direct transformation.
 ModeAB=           4 MOrb=             3 LenV=    1048222337
 LASXX=       222100 LTotXX=      222100 LenRXX=      452040
 LTotAB=      229940 MaxLAS=      459765 LenRXY=           0
 NonZer=      674140 LenScr=     1571840 LnRSAI=      459765
 LnScr1=     1378304 LExtra=           0 Total=      3861949
 MaxDsk=          -1 SrtSym=           T ITran=            4
 JobTyp=0 Pass  1:  I=   1 to   3.
 (rs|ai) integrals will be sorted in core.
 Spin components of T(2) and E(2):
     alpha-alpha T2 =       0.3582142781D-02 E2=     -0.1090222112D-01
     alpha-beta  T2 =       0.5062800759D-01 E2=     -0.1352708882D+00
     beta-beta   T2 =       0.3582142781D-02 E2=     -0.1090222112D-01
 ANorm=    0.1028490298D+01
 E2 =    -0.1570753304D+00 EUMP2 =    -0.38945660778297D+02
 Keep R2 and R3 ints in memory in symmetry-blocked form, NReq=8950725.
 Iterations= 100 Convergence= 0.100D-06
 Iteration Nr.   1
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        12
 NAB=     6 NAA=     0 NBB=     0.
 MP4(R+Q)=  0.44201243D-02
 E3=       -0.26157475D-01        EUMP3=      -0.38971818253D+02
 E4(DQ)=   -0.64973377D-02        UMP4(DQ)=   -0.38978315591D+02
 E4(SDQ)=  -0.70395934D-02        UMP4(SDQ)=  -0.38978857846D+02
 DE(Corr)= -0.17879850     E(Corr)=     -38.967383953    
 NORM(A)=   0.10472750D+01
 Iteration Nr.   2
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        12
 NAB=     6 NAA=     0 NBB=     0.
 DE(Corr)= -0.18821402     E(CORR)=     -38.976799463     Delta=-9.42D-03
 NORM(A)=   0.10722546D+01
 Iteration Nr.   3
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        12
 NAB=     6 NAA=     0 NBB=     0.
 DE(Corr)= -0.19548669     E(CORR)=     -38.984072134     Delta=-7.27D-03
 NORM(A)=   0.11062781D+01
 Iteration Nr.   4
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        12
 NAB=     6 NAA=     0 NBB=     0.
 DE(Corr)= -0.20113762     E(CORR)=     -38.989723067     Delta=-5.65D-03
 NORM(A)=   0.11285457D+01
 Iteration Nr.   5
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        12
 NAB=     6 NAA=     0 NBB=     0.
 DE(Corr)= -0.20319733     E(CORR)=     -38.991782779     Delta=-2.06D-03
 NORM(A)=   0.11328197D+01
 Iteration Nr.   6
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        12
 NAB=     6 NAA=     0 NBB=     0.
 DE(Corr)= -0.20356268     E(CORR)=     -38.992148127     Delta=-3.65D-04
 NORM(A)=   0.11332830D+01
 Iteration Nr.   7
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        12
 NAB=     6 NAA=     0 NBB=     0.
 DE(Corr)= -0.20361026     E(CORR)=     -38.992195707     Delta=-4.76D-05
 NORM(A)=   0.11332728D+01
 Iteration Nr.   8
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        12
 NAB=     6 NAA=     0 NBB=     0.
 DE(Corr)= -0.20361041     E(CORR)=     -38.992195860     Delta=-1.53D-07
 NORM(A)=   0.11332641D+01
 Iteration Nr.   9
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        12
 NAB=     6 NAA=     0 NBB=     0.
 DE(Corr)= -0.20360865     E(CORR)=     -38.992194101     Delta= 1.76D-06
 NORM(A)=   0.11332649D+01
 Iteration Nr.  10
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        12
 NAB=     6 NAA=     0 NBB=     0.
 DE(Corr)= -0.20360839     E(CORR)=     -38.992193840     Delta= 2.60D-07
 NORM(A)=   0.11332637D+01
 Iteration Nr.  11
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        12
 NAB=     6 NAA=     0 NBB=     0.
 DE(Corr)= -0.20360820     E(CORR)=     -38.992193645     Delta= 1.95D-07
 NORM(A)=   0.11332627D+01
 Iteration Nr.  12
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        12
 NAB=     6 NAA=     0 NBB=     0.
 DE(Corr)= -0.20360808     E(CORR)=     -38.992193524     Delta= 1.21D-07
 NORM(A)=   0.11332619D+01
 Iteration Nr.  13
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        12
 NAB=     6 NAA=     0 NBB=     0.
 DE(Corr)= -0.20360800     E(CORR)=     -38.992193443     Delta= 8.13D-08
 NORM(A)=   0.11332615D+01
 Dominant configurations:
 ***********************
 Spin Case        I    J    A    B          Value
   ABAB           3    3    5    5      -0.320397D+00
   ABAB           4    4    5    5      -0.320397D+00
 Largest amplitude= 3.20D-01
 Time for triples=        2.93 seconds.
 T4(CCSD)= -0.13230109D-01
 T5(CCSD)=  0.19026724D-04
 CCSD(T)= -0.39005404525D+02
 Discarding MO integrals.

 **********************************************************************

            Population analysis using the SCF density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A1) (?A) (?B) (?B)
       Virtual   (?B) (?A) (?B) (?B) (?B) (?B) (?B) (?B) (?A) (?A)
                 (?A) (?B) (?B) (?B) (?A) (?B) (?B) (?B) (?B) (?B)
                 (?B) (?B) (?B) (?B) (?B) (?B) (?B) (?A) (?A) (?A)
                 (?B) (?B) (?B) (?A) (?B) (?B) (?B) (?A) (?B) (?A)
                 (?B) (?B) (?B) (?B) (?B) (?B) (?B) (?B) (?B) (?B)
                 (?B) (?A) (?A) (?A) (?B) (?B) (?B) (?A) (?A) (?B)
                 (?B) (?B) (?B) (?B) (?B) (?A) (?B) (?B) (?B) (?B)
                 (?B) (?B) (?B) (?B) (?B) (?A) (?A) (?A) (?B) (?B)
                 (?B) (A1)
 Unable to determine electronic state:  an orbital has unidentified symmetry.
 Alpha  occ. eigenvalues --  -12.04404  -1.68459  -1.29896  -1.29896
 Alpha virt. eigenvalues --   -0.88999  -0.34731  -0.29440  -0.29440  -0.28378
 Alpha virt. eigenvalues --   -0.19281  -0.18191  -0.18191   0.01567   0.02146
 Alpha virt. eigenvalues --    0.05139   0.05400   0.05400   0.06183   0.16067
 Alpha virt. eigenvalues --    0.43407   0.43407   0.43751   0.50462   0.50462
 Alpha virt. eigenvalues --    0.51692   0.68248   0.69404   0.69404   0.89905
 Alpha virt. eigenvalues --    0.90031   0.90031   0.93696   0.95477   1.11082
 Alpha virt. eigenvalues --    1.67456   1.68929   1.68929   1.83491   2.23069
 Alpha virt. eigenvalues --    2.23433   2.23433   2.35177   2.35984   2.36572
 Alpha virt. eigenvalues --    2.36692   2.36692   2.53944   2.54517   2.54517
 Alpha virt. eigenvalues --    2.85866   2.85866   2.88199   2.98780   2.98780
 Alpha virt. eigenvalues --    2.98968   3.05208   3.06191   3.11873   3.33880
 Alpha virt. eigenvalues --    3.33880   3.34971   3.37056   3.37692   3.65120
 Alpha virt. eigenvalues --    3.65120   3.65647   3.76864   3.79288   3.79288
 Alpha virt. eigenvalues --    4.02998   4.28294   4.30237   4.30237   4.70448
 Alpha virt. eigenvalues --    4.70611   4.70611   4.72680   4.73153   4.73153
 Alpha virt. eigenvalues --    4.77906   4.84484   4.87333   5.40278   5.40278
 Alpha virt. eigenvalues --    5.42346  13.62933
          Condensed to atoms (all electrons):
               1          2          3          4          5
     1  C    4.646726   0.289821   0.289821   0.289821   0.289821
     2  H    0.289821   0.264169   0.007307  -0.020106   0.007307
     3  H    0.289821   0.007307   0.264169   0.007307  -0.020106
     4  H    0.289821  -0.020106   0.007307   0.264169   0.007307
     5  H    0.289821   0.007307  -0.020106   0.007307   0.264169
 Mulliken charges:
               1
     1  C    0.193992
     2  H    0.451502
     3  H    0.451502
     4  H    0.451502
     5  H    0.451502
 Sum of Mulliken charges =   2.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C    2.000000
 Electronic spatial extent (au):  <R**2>=             20.9576
 Charge=              2.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=              0.0000    Y=              0.0000    Z=              0.0000  Tot=              0.0000
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             -2.5875   YY=             -0.0396   ZZ=             -2.5875
   XY=              0.0000   XZ=              0.0000   YZ=              0.0000
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             -0.8493   YY=              1.6986   ZZ=             -0.8493
   XY=              0.0000   XZ=              0.0000   YZ=              0.0000
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=              0.0000  YYY=              0.0000  ZZZ=              0.0000  XYY=              0.0000
  XXY=              0.0000  XXZ=              0.0000  XZZ=              0.0000  YZZ=              0.0000
  YYZ=              0.0000  XYZ=              2.8773
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=             -6.5854 YYYY=             -3.0276 ZZZZ=             -6.5854 XXXY=              0.0000
 XXXZ=              0.0000 YYYX=              0.0000 YYYZ=              0.0000 ZZZX=              0.0000
 ZZZY=              0.0000 XXYY=             -0.2083 XXZZ=             -0.8525 YYZZ=             -0.2083
 XXYZ=              0.0000 YYXZ=              0.0000 ZZXY=              0.0000
 N-N= 1.339224679282D+01 E-N=-1.112417588149D+02  KE= 3.953318474127D+01
 Symmetry A    KE= 3.487475125316D+01
 Symmetry B1   KE= 2.329216744052D+00
 Symmetry B2   KE= 1.282638165297D-17
 Symmetry B3   KE= 2.329216744052D+00
 1\1\GINC-GAMBA\SP\RCCSD(T)-FC\Gen\C1H4(2+)\IRIKURA\17-May-2017\0\\# CC
 SD(T,maxcyc=100)/GEN gfinput geom=check scf=xqc\\BEB step 10: low-spin
  CCSD(T) for dication of ch4\\2,1\C,0,0.,0.000000001,0.000000001\H,0,-
 0.000000006,-0.0000000007,1.0935113774\H,0,0.,-1.0309724112,-0.3645037
 927\H,0,-0.892848297,0.5154862096,-0.3645037952\H,0,0.892848303,0.5154
 862062,-0.3645037855\\Version=ES64L-G09RevD.01\HF=-38.7885854\MP2=-38.
 9456608\MP3=-38.9718183\MP4D=-38.9827357\MP4DQ=-38.9783156\MP4SDQ=-38.
 9788578\CCSD=-38.9921934\CCSD(T)=-39.0054045\RMSD=8.388e-10\PG=TD [O(C
 1),4C3(H1)]\\@


    UPON JULIA'S CLOTHES

 WHENAS IN SILKS MY JULIA GOES,
 THEN, THEN, METHINKS, HOW SWEETLY FLOWS
 THAT LIQUEFACTION OF HER CLOTHES.

 NEXT, WHEN I CAST MINE EYES, AND SEE
 THAT BRAVE VIBRATION, EACH WAY FREE,
 O, HOW THAT GLITTERING TAKETH ME!
     -- ROBERT HERRICK, 1648
 Job cpu time:       0 days  0 hours  0 minutes 27.9 seconds.
 File lengths (MBytes):  RWF=     38 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 09 at Wed May 17 13:21:37 2017.
